appserver/resources/mail/mail-connector/src/main/java/org/glassfish/resources/mail/admin/cli/CreateMailResource.java

```diff
@@ -45,17 +45,17 @@
 @RestEndpoints({
         @RestEndpoint(configBean=Resources.class,
                 opType=RestEndpoint.OpType.POST,
-                path="create-javamail-resource",
-                description="create-javamail-resource")
+                path="create-mail-resource",
+                description="create-mail-resource")
 })
 @org.glassfish.api.admin.ExecuteOn(RuntimeType.ALL)
-@Service(name="create-javamail-resource")
+@Service(name="create-mail-resource")
```

Comment:
This is a rename of an asadmin command so will need documentation updates

Reply:
Yes, I believe I updated all the documentation in this repository.
Where do I find any other documentation that would need to be updated?

How are we collecting information for "release notes"?  Or are we not doing that?

Reply:
Ideally we would be disciplined and create an issue for our change and give it a milestone in GitHub as that is the only issue tracker we have. That way we can work out our release notes. I will raise this conversation to the list.